uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -0,0 +1,254 @@
+
+/*******************************************************************************
+ *  Copyright 2012-2022 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+import Esri.ArcGISRuntime.Toolkit.Controller 100.14
+import QtQuick 2.12
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+
+/*!
+  \qmltype FloorFilter
+  \inqmlmodule Esri.ArcGISRuntime.Toolkit
+  \since 100.14
+  \brief Allows to display and filter the available floor aware layers in the current \c GeoView.
+*/
+Item {
+    id: floorFilter
+    property var geoView
+
+    property FloorFilterController controller: FloorFilterController {}
+
+    Binding {
+        target: controller
+        property: "geoView"
+        value: floorFilter.geoView
+    }
+
+    GridLayout {
+        columns: 2
+        ColumnLayout {
+            Layout.alignment: Qt.AlignBottom
+            ToolBar {
+                id: levelFilterMenu
+                visible: !internal.isFloorFilterCollapsed
+                // property var itemSelected: {text: ""}
+                property string itemSelected: ""
+
+                Action {
+                    id: close
+                    icon.source: "images/x.svg"
+                    onTriggered: {
+                        internal.isFloorFilterCollapsed = true
+                    }
+                }
+
+                ColumnLayout {
```

Comment:
❓   Is this `ColumnLayout` required? I notice the other toolbar has no internal `ColumnLayout`, I assume because of the outer `ColumnLayout`.

Reply:
The line 59 `ColumnLayout` is used to contain all the `ToolsBars` and they will expand the Layout. 
The same behaviour could be acheived with an `Item` but  then is needed to be set a `width` and `height` on it so it actually takes space (xes: `width: childRect.width`). What do you prefer?

Reply:
The other `ToolBar` do not have any Layouts because they are only a single item, so they will not overlap.
From the docs:

_ToolBar does not provide a layout of its own, but requires you to position its contents, for instance by creating a RowLayout. If only a single item is used within the ToolBar, it will resize to fit the implicit size of its contained item. This makes it particularly suitable for use together with layouts._

Long story short: multiple item `ToolBar` needs a Layout to not overlap their items.